app/src/main/java/org/greenstand/android/TreeTracker/models/TreeTrackerViewModelFactory.kt

```diff
@@ -0,0 +1,29 @@
+package org.greenstand.android.TreeTracker.models
+
+import android.content.res.Resources
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import java.lang.RuntimeException
+import org.greenstand.android.TreeTracker.dashboard.DashboardViewModel
+import org.greenstand.android.TreeTracker.languagepicker.LanguagePickerViewModel
+import org.greenstand.android.TreeTracker.preferences.PreferencesMigrator
+import org.greenstand.android.TreeTracker.signup.SignupViewModel
+import org.greenstand.android.TreeTracker.splash.SplashScreenViewModel
+
+@Suppress("UNCHECKED_CAST")
```

Comment:
I think this suppression can be removed by using `reified` but I can look into that later

Reply:
I don't think reified can be used on classes or override methods

Reply:
Ah okay dang. I hate this warning, but I haven't figured out how to remove it